master/builder_utils.py

```diff
@@ -127,11 +128,14 @@ def render_upload_command(props_obj):
     upload_filename = props_obj.getProperty("upload_filename")
     upload_tarball_name = props_obj.getProperty("upload_tarball_name")
     upload_tarball_path = upload_path.replace(upload_filename, upload_tarball_name)
+    upload_zip_name = props_obj.getProperty("upload_zip_name")
+    upload_zip_path = upload_path.replace(upload_filename, upload_zip_name)
     return ["sh", "-c",
         "[ '%s' != '%s' ] && aws s3 cp --acl public-read /tmp/julia_package/%s s3://%s ;"%(upload_filename, upload_tarball_name, upload_tarball_name, upload_tarball_path) +
+        "[ '%s' != '%s' ] && aws s3 cp --acl public-read /tmp/julia_package/%s s3://%s ;"%(upload_filename, upload_zip_name, upload_zip_name, upload_zip_path) +
```

Comment:
not really sure hot to make this a conditional

Reply:
I was thinking something like:

```
zip_upload_cmd = ""
if is_windows(props_obj):
    zip_upload_cmd = "aws s3 cp --acl public-read /tmp/julia_package/%s s3://%s ;"
```

Then, in the return statement, we can just put a `+ zip_upload_cmd` in.

Also, you don't need the `[ '%s' != '%s' ]` comparison; that check is just to ensure that we don't accidentally upload a tarball twice on Linux, since on Linux we only generate tarballs, whereas on Mac and Windows we generate something else (`upload_filename`, which is a `.dmg` or `.exe`) as well as a tarball.